test/controllers/address/PostcodeLookupControllerSpec.scala

```diff
@@ -71,7 +71,7 @@ class PostcodeLookupControllerSpec extends AddressBaseSpec {
 
     "return 200 if the user has entered a residency choice on the previous page" in new LocalSetup {
       override def sessionCacheResponse: Option[CacheMap] =
-        Some(CacheMap("id", Map("soleResidencyChoiceDto" -> Json.toJson(ResidencyChoiceDto(SoleAddrType)))))
+        Some(CacheMap("id", Map("addressPageVisitedDto" -> Json.toJson(AddressPageVisitedDto(true)))))
```

Comment:
what happens if this is false?

Reply:
It will do the same thing as if it's true as there's a block that checks for Some/None rather than the actual Boolean value.

Adding test anyway

Reply:
Done